cumulus/primitives/parachain-inherent/src/lib.rs

```diff
@@ -69,6 +110,22 @@ pub struct ParachainInherentData {
 	/// were sent. In combination with the rule of no more than one message in a channel per block,
 	/// this means `sent_at` is **strictly** greater than the previous one (if any).
 	pub horizontal_messages: BTreeMap<ParaId, Vec<InboundHrmpMessage>>,
+	/// Represents the relay parent block and its descendants.
+	/// This information is used to ensure that a parachain node builds blocks
+	/// at a specified offset from the chain tip rather than directly at the tip.
```

Comment:
If my understanding is correct:
- Would be nice to clarify that parent header must always be included, and number of descendants depends on the relay-parent-offset. Such comment may help to connect things.